src/test/java/uk/gov/companieshouse/overseasentitiesapi/service/OverseasEntitiesServiceTest.java

```diff
@@ -116,10 +116,12 @@ private void testSubmissionCreation(boolean isResumeLinkExpected, String entityN
 
         var trustDataDto = new TrustDataDto();
         trustDataDto.setTrustName("dto");
+        trustDataDto.setTrustName("ch_reference - dto");
```

Comment:
This line overwrites the value set on the line above, so wondering what the reason for doing it is? Same on line 124.

Reply:
copy paste 🤦‍♂️

Reply:
in reality, it is not necessary
I will remove it